### PR TITLE
Updated guidance on continuation prompts

### PR DESCRIPTION
**Pull request by julian-cable**
Updated to reflect the latest guidance on continuation prompts as communicated by Pat. A previous choice of three possible options is now replaced with just a single approach.

@daobrien @sffrench Please review.

### CODE REVIEW

**Comment by daobrien** [comment]
@bonnevil @mweetman-redhat @alexpdp7 I know this is probably a "not again" topic for numerous ppl, but this is supposed to be the final word, and it needs more than just another editor to say "LGTM". I'd really appreciate at least one of you (I've invited Fernando but he hasn't accepted the invite yet) giving this the once-over and ack. Thanks.

**Review by sffrench — APPROVED** [review]
I am happy with this Julian. It is the agreed upon standard and I am happy we are all doing this now.

**Comment by bonnevil** [comment]
I'm not reviewing this, it's literally not my business at this point.

**Pull request merged**
